app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,68 @@
+class TasksController < ApplicationController
+  # RENAME FOLDER TASKS & change folder/controller associations
+  before_action :find_task, only: [:show, :edit, :update, :complete, :delete]
+
+  def index
+    @tasks = Task.all
+    @all_the_people = Person.all
+  end
+
+  def show
+    @all_the_people = Person.all
+  end
+
+  def new
+    @task = Task.new
+    @all_the_people = Person.all
+    @url = "/tasks/new"
+  end
+
+  def create
+    @task = Task.create(create_params)
+    redirect_to "/"
+  end
+
+  def complete
+      @task.completed = "Yes"
```

Comment:
You can also specify these things in the update method and that would combine these three lines into one line